src/assets/style.css

```diff
@@ -55,6 +57,20 @@
 
 /* Resets */
 
+.sr-only {
```

Comment:
Could you please add a comment here that says what sr-only means?

Reply:
added a comment